input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
Coming back to this: there are several publication where e_ref is not given, but differential fluxes are quoted in tables between e_min and e_max. Can't we add flexibility that in this case, e_ref is calculated dynamically? My feeling is that we add here information to the catalogue, which is not part of the publication.

Would have to discuss of course what to do:
- simple mean value
- weighted mean (with weights calculated from shape of the spectrum)
- ...